vendors/cypress/boards/CYW943907AEVAL1F/aws_demos/config_files/FreeRTOSIPConfig.h

```diff
@@ -73,15 +73,9 @@
  * socket has been destroyed, the result will be stored into the cache.  The next
  * call to FreeRTOS_gethostbyname() will return immediately, without even creating
  * a socket.
- *
- * NOTE: Use caution when enabling the DNS cache and connecting to services which
- * use a load balancer.  Since the cache only holds a single IP address, if that
- * address is out of date, no connections to the host will succeed until the TTL
- * expires.  See this GitHub issue for more details.
- *
- * https://github.com/FreeRTOS/FreeRTOS/issues/58
  */
```

Comment:
Do we want to have a similar note to this for a scenario where ipconfigUSE_DNS_CACHE == 1 and ipconfigDNS_CACHE_ADDRESSES_PER_ENTRY  == 1? This seems like it would have the same issue that this note addresses.

Reply:
It would, that's a good point.  I'll add a note in the example FreeRTOSIPConfig.h about this.

Reply:
Added a clarifying comment in FreeRTOSIPConfigDefaults.h